### ISSUE TITLE
[Discussion] General questions

### ISSUE THREAD

**Issue by thiagomurakami** [vtex/styleguide#511]
Some questions regarding the purpose of this project that I couldn't find the answer in the current documentation:

### How do we know if a component should be included in the `styleguide`?

Currently, my answer to this question is: if the component is used on more than one (VTEX) project, it's "eligible" to be part of the styleguide. I think the criteria should be really explicit.

### How complex can the components be?

I'll use [`Atomic Design`](http://atomicdesign.bradfrost.com/chapter-2/) to better formulate this question. Should components in `styleguide` be just `atoms`? Or are higher levels of complexity allowed? If they are, how should we present it?

### As a developer, I want to know the development principles and patterns used when using these components. Should this be in the documentation? Or is this outside the scope?

For example, we have all form components to build a form, however, we don't have any guideline on actually implementing a form. 

### Basically, what are the principles of this project? What does it solve?

In the home, we have the following phrase:

> This is the home for **all** product design related reusable patterns, components and assets.

However, at the same time, we have another VTEX components library ([VTEX Butik](https://butik.vtex.com/)).

**Comment by cmdalbem** [comment]
Related to https://github.com/vtex/styleguide/issues/503

**Comment by cmdalbem** [comment]
Work in progress: https://docs.google.com/document/d/1QzN5yMT-M_mlxEsiefPXYIVh3PhKjJVHHlsgWN_pePE/edit?usp=sharing

**Comment by thiagomurakami** [comment]
Mostly answered by the document 😄
